### PR TITLE
Update some cded keys in loaders

### PR DESCRIPTION
**Pull request by gigxz**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

A few fixes in the loaders. I compared them to CustomDataElementDefinitions present in the environment.

## Type of change
- [x] Bug fix

## Checklist before requesting review
- [ ] I have performed a self-review of my code
- [ ] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [ ] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [ ] My code follows the style guidelines of this project (rubocop)
- [ ] I have updated the documentation (or not applicable)
- [ ] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/mhmr_case_management_note_loader.rb [hunk 1]
```diff
@@ -33,7 +33,6 @@ class MhmrCaseManagementNoteLoader < CustomAssessmentLoader
       { key: 'mhmr_plan_to_proceed_resolution', label: 'Resolution of current need presented (explain):', field_type: 'string', repeats: false },
       { key: 'mhmr_progress_towards_goals', label: 'Progress lack of progress narrative', field_type: 'string', repeats: false },
       { key: 'mhmr_no_show', label: 'No show', field_type: 'string', repeats: false },
-      { key: 'mhmr_staff_signature', label: 'Staff Signature / Credentials / ID#:', field_type: 'string', repeats: false },
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/mhmr_case_management_note_loader.rb:36]: this question was removed

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/spdat_loader.rb [hunk 2]
```diff
@@ -9,7 +9,6 @@ class SpdatLoader < CustomAssessmentLoader
     ASSESSMENT_DATE_COL = 'Date Taken'.freeze
 
     CDED_CONFIGS = [
-      { label: 'Date Taken', key: 'spdat_date_taken', repeats: false, field_type: 'date' },
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/spdat_loader.rb:12]: This is stored as the AssessmentDate

File: drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/uha_loader.rb [hunk 3]
```diff
@@ -19,6 +19,7 @@ class UhaLoader < CustomAssessmentLoader
         { element_id: 12281, label: 'County', key: 'uha_alt_mail_address_county', repeats: false, field_type: 'string' },
         { element_id: 12282, label: 'Zip Code', key: 'uha_alt_mail_zip_code', repeats: false, field_type: 'string' },
         { element_id: 12283, label: 'Alternate Phone Number', key: 'uha_alt_phone', repeats: false, field_type: 'string' },
+        { element_id: 12284, label: 'Disabled', key: 'uha_disabled', repeats: false, field_type: 'boolean' },
```
Comment (gigxz) [inline drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/uha_loader.rb:22]: This question was missing. Element id is from the excel file.

**Review by ttoomey — APPROVED** [review]
Looks good, thanks

**Pull request merged**